.github/workflows/js_tests.yml

```diff
@@ -0,0 +1,35 @@
+name: Javascript wallet
```

Comment:
I'd also suggest having an explicit name for the job, something like "JavaScript Wallet Tests." and the same for the workflow file name.